posthog/management/commands/generate_demo_data.py

```diff
@@ -263,5 +287,35 @@ def materialize_common_columns(self) -> None:
                 )
                 for prop in sorted(person_properties)
             ],
-            backfill_period_days=365,
+            backfill_period_days=backfill_days,
         )
+
+    def initialize_dagster_materialization(self, backfill_days: int):
+        # Use GraphQL to connect to dagster development server.
+        # I tried a=some other approaches, i.e. CLI and python client, but these were both extremely slow and spun
+        # up a new instance of the Dagster code server for each request, which is not what we want. This API returns
+        # immediately and is non-blocking.
+        client = DagsterGraphQLClient(DAGSTER_UI_HOST, port_number=DAGSTER_UI_PORT)
```

Comment:
This is nice. We can check if we can do a whole group materialization for all partitions instead of iterating the dates.